src/App.tsx

```diff
@@ -4,7 +4,6 @@ import { HashRouter, Route, Routes } from 'react-router-dom'
 import { ToastContainer as PrestyledToastContainer } from 'react-toastify'
 import { Provider } from 'react-redux'
 import store from '$/store'
-import WalletIntegrationObserver from '$/components/WalletIntegrationObserver'
```

Comment:
See `useEagerConnectEffect` for details. The new hook replaces `WalletIntegrationObserver`.